nrf-hal-common/src/gpiote.rs

```diff
@@ -257,6 +259,15 @@ fn config_channel_event_pin<P: GpioteInputPin>(
             EventPolarity::None => w.mode().event().polarity().none(),
             EventPolarity::Toggle => w.mode().event().polarity().toggle(),
         };
+
+        cfg_if! {
```

Comment:
You don't need this macro here, the cfg attribute can just be applies to a block

Reply:
thank you, I'll adjust this. Does this add some overhead?

Reply:
No. cfg_if already expands to it anyways.

Reply:
I was not aware that attributes can also be applied to blocks.